server/migrations/20190404145044-create-comments.js

```diff
@@ -37,6 +37,6 @@ module.exports = {
     });
   },
   down: (queryInterface, Sequelize) => {
-    return queryInterface.dropTable('Comment');
+    return queryInterface.dropTable('Comments');
```

Comment:
Why this change to `Comments`? Kindly check the codebase and verify what is appropriate

Reply:
@paradisekelechi It is the migrations file, by following sequelize convention migrations are stated in plural.

Reply:
@davidokonji That's correct!